### PR TITLE
Logging using logrus

### LINKED ISSUE #15: Logging
Use a proper logger throughout the project (logrus?) so that we can log at various levels

### PR DESCRIPTION
**Pull request by jdonszelmann**
Closes #15 

---
~~Depends on #13~~

### CODE REVIEW

File: control_server/main.go [hunk 1]
```diff
@@ -3,26 +3,35 @@ package main
...
 )
 
 var (
 	static = flag.String("static", "control_server/static", "Static file dir to server under /static/")
 )
 
+func init() {
+	log.SetFormatter(&log.TextFormatter{ForceColors: true})
+	log.SetOutput(os.Stdout)
+	log.SetLevel(log.DebugLevel)
```
Comment (NULLx76) [inline control_server/main.go]: Maybe we should make log level configurable possibly using an environment variable?

**Comment by NULLx76** [comment]
After you fix linting and conflicts this can be merged

File: control_server/main.go [hunk 2]
```diff
@@ -32,7 +32,7 @@ func init() {
 	log.SetLevel(loglevel)
 
 	// log error after the logger is initialised
-	if err != nil && lvlstring != ""{
+	if err != nil && lvlstring != "" {
```
Comment (NULLx76) [inline control_server/main.go:35]: what if lvlstring _is_ empty and an error occured?

Comment (jdonszelmann) [reply control_server/main.go:35]: can't happen, caught earlier

**Review by NULLx76 — APPROVED** [review]

**Pull request merged**